package.json

```diff
@@ -1,132 +1,132 @@
-		"enzyme-to-json": "^3.3.5",
-		"es3ify-loader": "^0.2.0",
-		"eslint": "^5.12.0",
-		"eslint-config-prettier": "^3.3.0",
-		"eslint-loader": "^2.1.1",
-		"eslint-plugin-jsx-a11y": "^6.1.2",
-		"eslint-plugin-prettier": "^3.0.1",
-		"eslint-plugin-react": "^7.12.3",
-		"html-webpack-exclude-assets-plugin": "0.0.7",
-		"html-webpack-plugin": "^3.2.0",
-		"identity-obj-proxy": "^3.0.0",
-		"jest": "^24.0.0",
-		"jest-teamcity-reporter": "^0.9.0",
-		"mockdate": "^2.0.3",
-		"nerv-devtools": "^1.3.9",
-		"node-sass": "^4.11.0",
-		"pixrem": "^5.0.0",
-		"postcss-loader": "^3.0.0",
-		"prettier": "1.15.3",
-		"promise-polyfill": "^8.1.0",
-		"react": "^16.7.0",
-		"react-dom": "^16.7.0",
-		"sass-loader": "^7.1.0",
-		"source-map-loader": "^0.2.4",
-		"style-loader": "^0.23.1",
-		"stylelint": "^9.10.1",
-		"stylelint-config-standard": "^18.2.0",
-		"stylelint-order": "^2.0.0",
-		"stylelint-scss": "^3.5.4",
-		"stylelint-webpack-plugin": "^0.10.5",
-		"uglifyjs-webpack-plugin": "^2.1.2",
-		"webpack": "^4.29.5",
-		"webpack-cli": "^3.2.3",
-		"webpack-dev-server": "^3.2.1",
-		"webpack-merge": "^4.2.1"
-	},
-	"dependencies": {
-		"@babel/runtime": "^7.3.4",
-		"@nice-digital/browserslist-config": "^1.0.0",
-		"@nice-digital/icons": "^2.1.1",
-		"@nice-digital/nds-core": "1.0.4-alpha.0",
-		"accessible-autocomplete": "^1.6.2",
-		"classnames": "^2.2.6",
-		"es5-polyfill": "0.0.6",
-		"js-cookie": "^2.2.0",
-		"nervjs": "^1.3.9",
-		"prop-types": "^15.6.2",
-		"react-hot-loader": "^4.7.1",
-		"unfetch": "^4.1.0"
-	},
-	"files": [
-		"lib",
-		"*.d.ts"
-	],
-	"directories": {
-		"lib": "lib"
-	}
+  "name": "@nice-digital/global-nav",
+  "version": "1.0.0",
+  "description": "Global header and footer components for NICE digital services",
+  "main": "lib/index.js",
+  "types": "./types.d.ts",
+  "scripts": {
+    "// LOCAL DEV": "",
+    "start": "cross-env NODE_ENV=development webpack-dev-server --hot --inline",
+    "start:nohot": "cross-env HOT=false NODE_ENV=development webpack-dev-server",
+    "// PRODUCTION BUILDS": "",
+    "build": "cross-env NODE_ENV=production webpack --config webpack.config.prod.js",
+    "es5ify": "babel src -d lib --ignore \"src/*.test.js\",\"src/setupTests.js\",\"src/polyfill/**/*.js\"",
+    "copy": "copyfiles --up 1 src/**/*.{scss,json} lib",
+    "// TESTS": "",
+    "test": "npm run lint && npm run test:unit",
+    "test:unit": "jest --notify",
+    "test:unit:watch": "jest --watch",
+    "test:unit:coverage": "npm run test:unit -- --coverage",
+    "// LINTING": "",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "eslint \"**/*.{js,jsx}\"",
+    "lint:js:fix": "npm run lint:js -- --fix",
+    "lint:scss": "stylelint **/*.scss",
+    "lint:scss:fix": "npm run lint:scss -- --fix",
+    "// LIFECYCLE HOOKS": "",
+    "prepare": "npm run es5ify && npm run copy"
+  },
+  "author": "NICE Digital Services (https://github.com/nhsevidence/global-nav/graphs/contributors)",
```

Comment:
Could you replace the nhsevidence mentions whilst you're here please? Think there at least 3?